pages/_app.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect } from 'react'
 import App from 'next/app'
 import Head from 'next/head'
+import Script from 'next/script'
```

Comment:
I was getting a repeated error that `<script>` is no longer allowed in `next/head`.  Decided to fix this to quell those warnings by doing what it said (import `Script` and use that instead).